bower.json

```diff
@@ -10,10 +10,13 @@
     "angular-sanitize": "1.3.0-beta.11",
     "angular-resource": "1.3.0-beta.11",
     "angular-ui-router": "~0.2.10",
+    "bootstrap-sass": "~3.1.0",
     "hello": "~1.3.2",
     "ngCordova": "~0.1.11-alpha"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "platform": "~1.3.0"
```

Comment:
I'm not familiar with this package. What is it for? Why is it just a dev dependency instead of a main dependency?

Is [this its git repo?](https://github.com/polymer/platform), if so, looks like it's deprecated.